runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -101,4 +101,26 @@ default boolean notifyOnDuel()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		position = 6,
+		keyName = "ignoreSelfNotifications",
+		name = "Ignore own messages",
+		description = "Ignores your messages for notifications"
+	)
+	default boolean ignoreSelfNotifications()
```

Comment:
Should this even be an option? Seems unreasonable to even have possibility to notify yourself with own message

Reply:
Yeah, I was thinking it would be nice to test if it was working, but I get you. Removed with the newest commit.